pyproject.toml

```diff
@@ -36,11 +38,11 @@ dependencies = [
     # less than or equal to Python 3.10. In order to ensure we have
     # a consistent dependency closure across all Python versions,
     # we explicitly include ruamel.yaml.clib as a dependency.
-    "ruamel.yaml.clib>=0.2.0,<=0.2.7",
+    "ruamel.yaml.clib>=0.2.0,<=0.2.8",
```

Comment:
Why do we need to increase the ceiling?

Reply:
was facing a build issue with this library without this